src/util_i_varlists.nss

```diff
@@ -1066,6 +1108,114 @@ void SetListString(object oTarget, int nIndex, string sValue, string sListName =
         SetLocalString(oTarget, LIST_REF + sListName + IntToString(nIndex), sValue);
 }
 
+int CopyListFloat(object oSource, object oTarget, string sSourceName, string sTargetName, int nIndex, int bAddUnique = FALSE, int nRange = 1)
+{
+    float fValue;
+    int  i, nCopied, nCount = CountFloatList(oSource, sSourceName);
+
+    if (nIndex < 0 || nIndex > nCount || !nCount)

```

Comment:
The check to see if nIndex is within bounds should be `nIndex >= nCount` since the index is zero-based.

Reply:
Fixed in all functions and prototypes.